views/profile/index.hbs

```diff
@@ -1,5 +1,6 @@
 <html lang="en">
   <head>
+    <meta name="viewport" content="initial-scale=1.0, width=device-width" />
```

Comment:
is this so it works on mobile?

Reply:
Yes

Reply:
Yeah that's the meta tag for smaller screens, makes it look much better on mobile devices.